app/assets/stylesheets/pages.css.scss

```diff
@@ -876,3 +876,9 @@ a {
     margin-top: 20px;
   }
 }
+
+.example-application {
+  input[readonly], textarea[disabled] {
```

Comment:
I'm not quite sure what hound is saying here, this [should be valid](https://github.com/CSSLint/csslint/wiki/Disallow-unqualified-attribute-selectors)